### PR TITLE
fix for re-render issue

### PR DESCRIPTION
**Pull request by ghost**
#116 This worked for me. I'm no longer experiencing re-renders caused bu actions.

### CODE REVIEW

File: src/client/ui/preview.js [hunk 1]
```diff
@@ -26,11 +28,15 @@ export function renderMain(data) {
     return ReactDOM.render(noPreview, rootEl);
   }
 
-  // We need to unmount the existing set of components in the DOM node.
-  // Otherwise, React may not recrease instances for every story run.
-  // This could leads to issues like below:
-  //    https://github.com/kadirahq/react-storybook/issues/81
-  ReactDOM.unmountComponentAtNode(rootEl);
+  if (selectedKind !== previousKind || previousStory !== selectedStory) {
```
Comment (arunoda) [inline src/client/ui/preview.js]: Add a comment on why we do this and add the link to the related issue.

File: src/client/ui/preview.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ import {
 } from '../';
 
 const rootEl = document.getElementById('root');
+const previousKind = '';
```
Comment (arunoda) [inline src/client/ui/preview.js]: You need to make these as `let`. `const` variables cannot be re-assigned.

Comment (mnmtanish) [inline src/client/ui/preview.js]: Hi @okonuskan 
This should be a `let` right?

File: src/client/ui/preview.js (hunk 1 above)
Comment (okonuskan) [inline src/client/ui/preview.js]: Comment added

File: src/client/ui/preview.js (hunk 2 above)
Comment (okonuskan) [inline src/client/ui/preview.js]: Yes, thats right. It's fixed now.

**Comment by arunoda** [comment]
Thanks.

**Comment by arunoda** [comment]
Released with `v1.16.0`

**Pull request merged**